app/templates/error403.html

```diff
@@ -3,5 +3,5 @@
 {% block body_content %}
 <h1>Permission Denied</h1>
 <h5>Only full-time ITS employees have permission to access this page.</h5>
-<h5>Return to the <a href="{{ url_for('ShiftsView:index') }}">homepage</a>.</h5>
+<h5>Return to the <a href="{{ url_for('ShiftsView:cas_passthrough') }}">homepage</a>.</h5>
```

Comment:
is that the route you want to go to? Don't you want to go to the index?

Reply:
Whoops, yes I want to go to the index. Changing that back.